pagopa/io-app
# fix: [IABT-1373] Fix support CTA behaviour

## Short description
This PR fixes a wrong behavior in the support CTA.
Now the CTA are coordinated to show them together if they should be displayed.

To have more details check [the doc on confluence](https://pagopa.atlassian.net/wiki/spaces/IOAPP/pages/451051849/IO+App+-+Setup+Utilizzo#CTA-Flusso-assistenza).

## List of changes proposed in this pull request
- Refactored the `ZendeskSupportComponent` component


https://user-images.githubusercontent.com/11773070/173394760-ca82a88b-1e0b-4d53-916e-79658b268d4d.mp4



## How to test
Test suite is provided.
Try to open the support screen with the different combinations and check the CTA.
